OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj

```diff
@@ -46,6 +44,8 @@
     </CustomCommands>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <WarningLevel>4</WarningLevel>
+    <Optimize>false</Optimize>
```

Comment:
VS/XS/MD make these changes, but we do not want them.
@scshunt please only commit file additions/removals from csproj files.